src/prefect/cli/__init__.py

```diff
@@ -80,6 +84,24 @@ def cli():
 # Miscellaneous Commands
 
 
+@cli.command()
+def repl():
+    """
+    Start a REPL
+    """
+    banner = f"""
+Welcome to Prefect REPL
+version : {prefect.__version__}
+    """
+
+    prompt_kwargs = {
+        "history": FileHistory(os.path.expanduser("~/.prefect/repl-history")),
```

Comment:
`~/.prefect` should be loaded from the `home_dir` configuration value instead of being hard coded.